db/config.cc

```diff
@@ -931,6 +931,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , wasm_udf_total_fuel(this, "wasm_udf_total_fuel", value_status::Used, 100000000, "Wasmtime fuel a WASM UDF can consume before termination")
     , wasm_udf_memory_limit(this, "wasm_udf_memory_limit", value_status::Used, 2*1024*1024, "How much memory each WASM UDF can allocate at most")
     , relabel_config_file(this, "relabel_config_file", value_status::Used, "", "Optionally, read relabel config from file")
+    , object_storage_config_file(this, "object_storage_config_file", value_status::Used, "", "Optionally, read object-storage endpoints config from file")
```

Comment:
It should be liveupdateable, so a cluster can be retrofitted with object storage without restarting nodes.

Reply:
Should existing sstables that got their config entry be updated too?

Reply:
Ideally yes. So if someone wants to change a bucket from no-auth to auth, or rotate a key, they can.

Reply:
Mostly live-updateable in updated version.
Existing sstables continue using old creds, but if these are wrong it won't help until #13745 is fixed, will do it inrementally